ui/index.html

```diff
@@ -41,6 +42,9 @@
 			property="twitter:image"
 			content="https://www.binarytree.dev/bt.jpg"
 		/>
+		
+		<title>BinaryTree: Developer Productivity Tools</title>
```

Comment:
I think this line will stop, dynamic title change when the user navigates to a different page.